Source/WebCore/platform/win/ClipboardUtilitiesWin.cpp

```diff
@@ -302,7 +302,7 @@ void replaceNBSPWithSpace(String& str)
 {
     static const UChar NonBreakingSpaceCharacter = 0xA0;
     static const UChar SpaceCharacter = ' ';
-    str.replace(NonBreakingSpaceCharacter, SpaceCharacter);
+    makeStringByReplacingAll(str, NonBreakingSpaceCharacter, SpaceCharacter);
```

Comment:
I guess this is where the GTK code was copied and pasted from.

Reply:
Ouch, I am missing an `str = ` here too. I guess I should really add a WARN_UNUSED_RETURN